docs/conf.py

```diff
@@ -274,6 +274,10 @@ def setup(app):
 
 html_js_files = [
     'js/custom.js',
+    (
+        'https://views.scientific-python.org/js/plausible.js',
+        {"data-domain": "pyhf.readthedocs.io", "defer": "defer"},
```

Comment:
@jpivarski @agoose77 You used `awkward-array.org` for your data-domain in https://github.com/scikit-hep/awkward/pull/1822. Is that your GitHub pages or is that a custom domain that your RTD redirects to?

Reply:
Oh I guess neither as you deploy to AWS:

https://github.com/scikit-hep/awkward/blob/66b620958b9e0bd2f16787b66596a22249e17336/.github/workflows/docs.yml#L270-L285

Reply:
Right, we deploy to AWS (RTD is inconvenient to work with if you want to build on GHA)